README.md

```diff
@@ -152,4 +152,41 @@ What you need to do in that case is to use the `org.wildfly:wildfly-mcp-api` art
 You may want to take a look at [wildfly-weather](https://github.com/ehsavoie/wildfly-weather) example.
 
 You can then use [widldfly-mcp-chatbot](https://github.com/wildfly-extras/wildfly-mcp/tree/main/wildfly-chat-bot) from the [wildfly-mcp](https://github.com/wildfly-extras/wildfly-mcp) project to connect via Server-Sent-Event to it and play with your tools.
+### Securing the MCP Server
+
+To secure your MCP Server, you can configure the OIDC authentication mechanism. You can use Keycloak container image: 
```

Comment:
Maybe could specify that Bearer token authentication is being used via the support provided by the elytron-oidc-client subsystem